en/telemetry/sik_radio.md

```diff
@@ -0,0 +1,71 @@
+# SiK Radio
+
+[SiK radio](https://github.com/LorenzMeier/SiK) is a collection of firmware and tools for telemetry radios.
+
+Hardware for the SiK radio can be obtained from various manufacturers/stores in variants that support different range and form factors. Typically you will need a pair of devices - one for the vehicle and one for the ground station. The ground station-based radio is connected via USB (essentially plug-n-play). The vehicle is connected to the flight-controller's `TELEM` port, and typically requires no further configuration.
+
+### Vendors
+
+* 3DR Radio \(small\) - [US (915MHz)](https://store.3dr.com/products/915-mhz-telemetry-radio) / [European (433MHz)](https://store.3dr.com/products/433-mhz-telemetry-radio)
+* [HK Radio](http://www.hobbyking.com/hobbyking/store/uh_viewitem.asp?idproduct=55559) \(small\)
+* [RFD900u](http://rfdesign.com.au/products/rfd900u-modem/) \(small\)
+* [RFD900](http://rfdesign.com.au/products/rfd900-modem/) \(long range\)
+
+![SiK Radio](../../assets/hardware/telemetry/sik_radio.jpg)
+
+## Build Instructions
```

Comment:
@bkueng So I think that generally speaking if you purchase SiK hardware it will already contain firmware that is built and configured appropriately. Is that correct?
So perhaps best thing here is to add a note that you can build the source and configure the software, but generally speaking you won't need to?

Reply:
> So I think that generally speaking if you purchase SiK hardware it will already contain firmware that is built and configured appropriately

Yes. It's however possible that it does not yet support mavlink 2, and then you need to update.

> So perhaps best thing here is to add a note that you can build the source and configure the software, but generally speaking you won't need to?

Yes, and I've never even done that myself :) I don't think it needs to be part of the user-guide, maybe in the dev-guide. And more importantly: it's even possible to update the radio firmware via QGC (in the same way as updating the autopilot FW via QGC works: change to the FW upgrade tab, then attach the radio).

Reply:
>  I don't think it needs to be part of the user-guide

It belongs in the user guide because a user (someone who wants to build vehicles using existing supported components) might reasonably need to do this.  Agree it is not a "main use case", hence need note to make that clear. Thanks for info. Will update.

Reply:
Actually, I'm changing my mind on this. No user should have to see this :-)